react-native-fast-image.podspec

```diff
@@ -20,8 +20,5 @@ Pod::Spec.new do |s|
   s.exclude_files = "ios/Vendor/**/*.{h,m}"
 
   s.dependency 'React'
-  s.dependency 'SDWebImage/Core'
-  s.dependency 'SDWebImage/GIF'
-  s.dependency 'SDWebImage/WebP'
```

Comment:
In 5.0, all external image format (Not supported by Apple's built-in framework), is a coder plugin.

The user, can using another Pod, to register the plugin, don't need a podspec subspec for this. Which make it more modular and under control.

For example, if user want to use WebP format, they can do this:

```
pod 'react-native-fast-image', :path => '../node_modules/react-native-fast-image'
pod 'SDWebImageWebPCoder'
```

Then register the coder plugin, in `AppDelegate.m`

```objectivec
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
// ...
// Register WebP format support
[SDImageCodersManager.sharedManager addCoder:SDImageWebPCoder.sharedCoder]
// ...
}
```

Reply:
This would mean that by default this module would no longer support as many formats though right?

Reply:
> This would mean that by default this module would no longer support as many formats though right?

Whether SDWebImage supports some format, or not. It's now, totally controlled by the users. If users want, they can supports all the format, listed in : [Coder Plugin List](https://github.com/SDWebImage/SDWebImage/wiki/Coder-Plugin-List).

The main repo (or `SDWebImage` pod itself), only supports the built-in format by Apple's SDK. Typically the JPEG/PNG/TIFF/GIF/HEIF, etc.

And I see, react-native-fast-image SDK, didn't rely on any dependency on what original image format is. Isn't it ? We just care about the animation, and their meta info (loop count/ frame count), but not what the original compressed data is.